Code/GraphMol/catch_graphmol.cpp

```diff
@@ -2991,11 +2991,12 @@ TEST_CASE(
     std::random_device rd;
     std::mt19937 g(rd());
     for (int i = 0; i < 100; ++i) {
+      SmilesParserParams ps;
```

Comment:
Maybe this could go outside the loop?

Reply:
I think the possible performance hit is ok in testing code, otherwise I'd definitely agree